js/main.js

```diff
@@ -33,6 +33,36 @@ maps.forEach(map_ => {
               duration: 1.0
             });
         }, 1000)
+    
+
+        let tobacco_field_data = {
+            "Tobacco": { 
+               data: tobacco_fields
+            },
+            "Soybean": {
+                data: soybeans_feilds
+            },
+            "Maize": {
+                data: maize_feilds
+            }
+        }
+
+        let baseOne = {}, tobaccoField_obj = {}
```

Comment:
What are we using `tobaccoField_obj` for?

Reply:
I removed it, it was actually doing nothing.